tiledb/CMakeLists.txt

```diff
@@ -279,6 +334,14 @@ endif()
 if (TILEDB_SERIALIZATION)
   add_definitions(-DTILEDB_SERIALIZATION)
   message(STATUS "The TileDB library is compiled with query serialization enabled.")
+
+  if(MSVC)
+    if (TILEDB_CURL_EP_BUILT)
+      target_compile_definitions(TILEDB_CORE_OBJECTS PRIVATE -DCURL_STATICLIB=1)
+    else()
+      target_compile_definitions(TILEDB_CORE_OBJECTS PRIVATE                   )
```

Comment:
remove extra whitespace after `PRIVATE`

Reply:
removing entire else clause, no definitions to set (may have to revisit, environ with curl already installed in windows not tried)